Backpack/src/main/java/net/skyscanner/backpack/checkbox/BpkCheckbox.kt

```diff
@@ -30,27 +30,33 @@ open class BpkCheckbox @JvmOverloads constructor(
   }
 
   private fun initialize(attrs: AttributeSet?, defStyleAttr: Int) {
-    var defaultColor = ContextCompat.getColor(context, R.color.bpkTextSecondary)
-    var checkedColor = BpkTheme.getPrimaryColor(context)
-    var disabledColor = ContextCompat.getColor(context, R.color.bpkSkyGrayTint04)
+    var checkmarkDefaultColor = ContextCompat.getColor(context, R.color.bpkTextSecondary)
+    var checkmarkCheckedColor = BpkTheme.getPrimaryColor(context)
+    var checkmarkDisabledColor = ContextCompat.getColor(context, R.color.bpkSkyGrayTint04)
+    val textDisabledColor = ContextCompat.getColor(context, R.color.bpkTextTertiary)
```

Comment:
This seems a bit too dark for the text colour

Reply:
See the screenshots, this seems to be the only reasonable replacement among the semantic colours.

Reply:
I'm talking based on the screenshot, it doesn't look like a disabled text

Reply:
I think this should be `textQuaternary`. Is the same as tertiary now but that is wrong, see slack conversation for more.

Reply:
let's update this to `bpkSkyGrayTint04` for both dark and light mode, as this is what iOS is doing for link buttons for now

Reply:
Ok, the text colour is bpkSkyGrayTint04 now.